source/nodejs/adaptivecards-fabric/src/components/inputs/InputToggleFabric.tsx

```diff
@@ -12,9 +12,8 @@ export class InputToggleFabric extends Shared.ReactInputElement {
     private valueOff: string;
 
     public parse = (json: any, errors?: AC.IValidationError[]) => {
-        this.id = AC.getStringValue(json.id);
-        this.valueInternal = json.value;
-        this.defaultValue = this.valueInternal;
+        super.parse(json, errors);
+        this.value = AC.getStringValue(json.value, "");
         this.valueOn = AC.getStringValue(json.valueOn);
         this.valueOff = AC.getStringValue(json.valueOff);
         this.title = AC.getStringValue(json.title);
```

Comment:
You don't appear to be parsing/handling the `wrap` property.

Reply:
wrap is not supported in this version

Reply:
You mean in the Fabric control?